api/project_member.go

```diff
@@ -66,7 +66,8 @@ type ProjectMemberFind struct {
 	ID *int
 
 	// Related fields
-	ProjectID *int
+	ProjectID   *int
+	PrincipalID *int
```

Comment:
what's the use case?

will we find using both projectId and principalId? If we only use the latter, we cannot leverage the existing index. Since the index is UNIQUE(project_id, principal_id). it must start with project_id.